pyhuman/app/workflows/google_search.py

```diff
@@ -1,12 +1,13 @@
 from selenium.webdriver.common.keys import Keys
```

Comment:
Formatting nitpick, make sure to do imports from the standard library before imports from packages that need to be installed